### DISCUSSION TITLE
Chat here

### DISCUSSION THREAD

**Discussion by kokonut27** [JBYT27/REPLAPI#5] [General]
Hai. Chat here. If you're not a member, pls dont create ur own dicussion. Chat here. actually. lol, thanks.

**Comment by darkdarcool** [comment]
got it

**Comment by darkdarcool** [comment]
@JBYT27, is tomorrow a good time to test and push to pypI, this is coming along really good, and version 0.0.2 looks to be a good one! I also made release for version [here](https://github.com/JBYT27/REPLAPI/releases/tag/0.0.1)

**Reply by kokonut27** [reply]
ok, sure!

**Reply by darkdarcool** [reply]
noice!

**Comment by kokonut27** [comment]
Im gonna post beta now on replit ;)

**Reply by darkdarcool** [reply]
No!!!! we need to test first

**Reply by darkdarcool** [reply]
also you're a smart [swear word in a nice way] for getting the most recent post!

**Reply by kokonut27** [reply]
ok. and huh? wdym? oh. nvm

**Reply by darkdarcool** [reply]
we need to make sure everything works first!

**Reply by darkdarcool** [reply]
ill inv you to replit

**Reply by kokonut27** [reply]
ok

**Comment by darkdarcool** [comment]
@JBYT27, BTW, ill be on git for a while, so ping me here!

**Reply by kokonut27** [reply]
okie. its tmrw btw. like today is tmrw. lol

**Comment by darkdarcool** [comment]
So, Ima be creating docs for REPLAPI, and i want your feedback tomorrow!

**Reply by kokonut27** [reply]
okie

**Comment by kokonut27** [comment]
Hey @RayhanADev. I'm not sure you'll recieve ping, but instead of forking it, we wanted u to help. You can decline, but if you don't, thanks!

**Reply by darkdarcool** [reply]
lol im pretty sure he will

**Reply by whippingdot** [reply]
No he won't 😢

**Reply by darkdarcool** [reply]
lol

**Comment by darkdarcool** [comment]
@JBYT27, i created the 0.0.2 release [right here on this link](https://github.com/JBYT27/REPLAPI/releases/tag/0.0.2)

**Reply by kokonut27** [reply]
ok, thanks

**Comment by darkdarcool** [comment]
@JBYT27, icinghacks gave us this link for a faster way to look at cycles on other things? Can you integrate this as soon as you can?

https://replit.com/@IcynHackz/Python-Profile?v=1

**Reply by kokonut27** [reply]
ok, sure!

**Comment by kokonut27** [comment]
@darkdarcool!!!! Come to repl!

**Reply by darkdarcool** [reply]
sure

**Reply by kokonut27** [reply]
uh, can u go back on. to talk? about wat we're gonna do?

**Reply by kokonut27** [reply]
@darkdarcool, sry for ping, but r u on? repl? or github?

**Comment by darkdarcool** [comment]
@RayhanADev, we started using your source of info for replit(the thing you tried to do in python). We found a module called `repltalk` that does it so we use the same thing you do! Just letting you know!

Cheers!

**Comment by whippingdot** [comment]
OI! Did you know that you can disable Wikis? And Projects? You don't use either so I would recommend to disable those in options.

Also can you add me to tis project? I wanna help and learn how to do stuff loike this. I know both of you very well(@darkdarcool & @JBYT27) so...

EDIT: Umm if you add me I can help with the docs website(sorry to be mean but it kinda bad). I can do CSS and HTML for that. You guys don't seem to update it (it says the version is 0.0.3) so I can do that for you and the links aren't styled so I can do that too. Also when you press the darkmode toggle the picture does not change to a sun and the background color of the button doesn't change so...I can help with that too?

**Reply by darkdarcool** [reply]
we might use that in the future, but we dont have the energy to do it rn. And we'll add you. Just do stuff(it's on replit)

**Reply by darkdarcool** [reply]
https://replit.com/@darkdarcool/ReplAPI-Docs#README.md

**Reply by whippingdot** [reply]
ok, also add me to the repo BOI(if you don't have the perms i can wait for JB)

P.S. I will help on this but not today 4 sure. I have an IMPORTANT exam tomorrow so i can't. After tomorrow tho I have a 4 day break so maybe...

**Comment by darkdarcool** [comment]
@JBYT27, @RayhanADev, should me make a bin? It seems like a better way of doing things if we ever need to ask the user any questions

**Reply by whippingdot** [reply]
bin...

P.S. Still waiting on the decision...

**Reply by darkdarcool** [reply]
i think a bin may be a good idea

and I made my decision. I'm not going to invite you to the repo, but can you make pull requests? And maybe we'll inv you a couple times to the repl :))))   (leaning towards yes on the repo, but i'll need to talk it over with jb first)
